platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-peregrine.config

```diff
@@ -62,9 +62,9 @@ scripts=[\
 # Set Permissions for Sling Distribution
     set ACL for distribution-agent-user, defaultAgentService
         allow jcr:all on /content
-        allow jcr:all on /etc/distribution
-        allow jcr:all on /libs/sling/distribution
-        allow jcr:all on /var/sling/distribution
+#        allow jcr:all on /etc/distribution
+#        allow jcr:all on /libs/sling/distribution
+#        allow jcr:all on /var/sling/distribution
```

Comment:
I think we should revert this as I guess it provides a different fix from what we have in #757. @cmrockwell you are probably the most interested in `distribution` replication type and I see your approve here - should we maybe just remove the paths created here or rather go with #757?

Reply:
My approval here was a mistake. I went ahead and approved #757 but that only creates the paths so there are not errors when setting ACL's. With these ACL's removed. remote replication is broken.